third_party/mustache/README.amp

```diff
@@ -0,0 +1,10 @@
+URL: https://github.com/janl/mustache.js/
+License: The MIT License
+License File: https://github.com/janl/mustache.js/blob/master/LICENSE
+
+Description:
+Fork of mustache.js library.
+
+Local Modifications:
+None yet, but expected changes for restricting instructions and sanitizing
```

Comment:
@cramforce question, do we want to include this in third_party, or do like register-element and install through npm and import from node_modules/* ?